src/main/File/Services/DeleteFileService.java

```diff
@@ -0,0 +1,90 @@
+package File.Services;
+
+import Config.Message;
+import Config.Service;
+import Database.File.FileDao;
+import File.File;
+import File.FileMessage;
+import File.FileType;
+import User.UserType;
+import Validation.ValidationUtils;
+import com.mongodb.client.MongoDatabase;
+import org.bson.types.ObjectId;
+
+import java.util.Optional;
+
+public class DeleteFileService implements Service {
+  MongoDatabase db;
+  private FileDao fileDao;
+  private String username;
+  private String orgName;
+  private UserType userType;
+  private FileType fileType;
+  private String fileId;
+
+  public DeleteFileService(
+      MongoDatabase db,
```

Comment:
as stated before, lets remove the MongoDatabase db field and just rely on the DAO